src/jvm/test/org/jetbrains/kotlinx/lincheck_test/util/TestUtils.kt

```diff
@@ -227,5 +227,5 @@ class StringPoolGenerator(randomProvider: RandomProvider, configuration: String)
         strings[random.nextInt(strings.size)]
 }
 
-internal val OVERWRITE_REPRESENTATION_TESTS_OUTPUT: Boolean =
```

Comment:
If possible, let's keep it internal. However, it should not block the PR.

Reply:
Reverted.